package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "actions-project-version-check",
+  "version": "1.0.0",
+  "description": "GitHub Action that checks if the project version has been updated",
+  "main": "dist/index.js",
+  "scripts": {
+    "lint": "eslint src/index.js",
+    "package": "ncc build src/index.js --minify",
```

Comment:
Ziehen sich die Actions immer die aktuelle Version der dist/index.js?
Hier fehlt mir noch irgendwie die Information, wann jetzt wirklich ein "npm package" ausgeführt werden muss. Wäre irgendwie doof, wenn der Entwickler immer daran denken muss.

Reply:
Alternative ist, den Ordner "node_modules" zu pushen.

Reply:
Habe das jetzt gelöst, siehe "pre-commit" in der `package.json`